README.md

```diff
@@ -91,21 +91,21 @@ These are the supported specifications by this version.
 
 | **Specifications**                    | v0.1 | v0.2 | v0.3 | **v1.0** |
```

Comment:
We should just remove the column for v0.2. We already state we don't support 0.2 above.